modules/db/src/main/scala/gem/dao/SmartGcalDao.scala

```diff
@@ -31,6 +33,9 @@ object SmartGcalDao {
   def selectGmosNorth(k: SmartGcalKey.GmosNorthSearch, t: SmartGcalType): ConnectionIO[List[Int]] =
     t.fold(Statements.selectGmosNorthByLamp(k), Statements.selectGmosNorthByBaseline(k)).list
 
+  def selectGmosSouth(k: SmartGcalKey.GmosSouthSearch, t: SmartGcalType): ConnectionIO[List[Int]] =
+    t.fold(Statements.selectGmosSouthByLamp(k), Statements.selectGmosSouthByBaseline(k)).list
```

Comment:
There are two basic kinds of SmartGcal queries.  One looking for just a matching Arc or Flat specifically (by lamp) and one looking for day vs. night calibration sets.  The fold just picks the family of query that we will do.